modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
```

Comment:
Please add an explicit mention of the refresh_token scope, what it is, and how to use it to obtain new access tokens